autopush/senderids.py

```diff
@@ -78,17 +108,33 @@ def senderIDs(self):
             self._refresh()
         return self._senderIDs
 
+    def get(self, id):
+        """Return the associated record for a given SenderID"""
+        record = self._senderIDs.get(id)
+        if record:
+            record["senderID"] = id
+        return record
+
     def getID(self):
```

Comment:
nit: this isn't Python style, though unfortunately some libs we use have similar failings (ie, twisted). get_id would be pythonic. We might need to file a separate ticket to go through and thoroughly ensure we have a consistent style.